<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PureMediaServer</class>
 <widget class="QMainWindow" name="PureMediaServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample Media Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Pure Media Server 0.0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Media</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectPDButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect To PD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer1Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer2Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer3Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer4Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer5Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer6Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer7Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="layer8Check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer1Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer2Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer3Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer4Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer5Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer6Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer7Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="layer8Add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="readDMX">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read DMX</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="universe">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="winpositionx">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OLA Universe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Position</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="winpositiony">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="winsizey">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="winsizex">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Layer 8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="window">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>87</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Window</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
